schemas/stock_price_information.py

```diff
@@ -0,0 +1,10 @@
+from pydantic import BaseModel
+
+
+class StockPriceInformation(BaseModel):
+    provider: str
+    ticker: str
+    price: float
+
+    def __hash__(self):
+        return hash((type(self),) + tuple(self.__dict__.values()))
```

Comment:
Sample Schema with hash function that I found on Stack Overflow.

Used to compare `obj1 == obj2`